frontend/src/Editor/LeftSidebar/SidebarDebugger.jsx

```diff
@@ -56,20 +57,41 @@ export const LeftSidebarDebugger = ({ darkMode, components, errors }) => {
         })
     },[errors])
 
+    React.useEffect(() => {
+        const unReadErrors = errorLogs.length - unReadErrorCount.read
+
+        if(open ===  false && errorLogs.length !== unReadErrorCount.read) {
+            setUnReadErrorCount((prev) => {
+                let copy = JSON.parse(JSON.stringify(prev))
+                copy.unread = unReadErrors
+                return copy
+            })
+        }
```

Comment:
`unReadErrors` should be declared inside the if statement where its being used

Reply:
yes, i was using it in the other block too, but later changed and I guess did not notice, thanks